medicines/web/src/components/search-results/index.test.tsx

```diff
@@ -25,7 +25,7 @@ const drugsMock = [
   },
 ];
 
-const noFeedback = () => {};
+const noFeedback = () => undefined;
```

Comment:
Why is this better than `() => {}` here?

Reply:
I had to add a `// tslint:disable-next-line: no-empty` to this in order for it to commit; maybe `undefined` works around this?